xbmc/cores/VideoPlayer/DVDClock.h

```diff
@@ -21,7 +21,6 @@
  */
 
 #include "threads/CriticalSection.h"
-#include "TimingConstants.h"
```

Comment:
also this was initially PRed to vpupdates, but after you deleted the branch, it somehow took master as base and we didn't notice it.